src/js/controllers/tx-details.js

```diff
@@ -36,6 +54,69 @@ angular.module('copayApp.controllers').controller('txDetailsController', functio
     });
   });
 
+  function setLoading(name, state) {
+    $scope.loadingService[name] = state;
+    $timeout(function() {
+      $scope.$apply();
+    });
+  };
+
+  function verifyByThirdParties() {
+    if (lodash.isEmpty($scope.availableServices) || $scope.wallet.network == 'testnet') return;
+
+    lodash.each($scope.availableServices, function(service) {
+      setLoading(service.name, true);
+
+      verifyByThirdPartyService.getTx(service, txId, function(err, params) {
```

Comment:
The service api should be something like:


thirdPartyExplorers.verify(tx, function(err, result) {
   if (err) return ...;

   $scope.verifiedOK = result.ok;
   $scope.verifiedNOK = result.nok;
   $scope.verifiedNA = result.na;
});